### DISCUSSION TITLE
Stop searching for same torrents after certain time

### DISCUSSION THREAD

**Discussion by ajhorvath20** [cross-seed/cross-seed#85] [Q&A] [answered]
Hi, I'm sure there is an option which I'm not aware of, is it possible to tell cross-seed to stop searching for same torrents after a week. 

Or avoid searching the tracker when file already saved or injected.

I'm using `command: search`

Thank you

**Comment by mmgoodnow** [comment, marked as answer]
~There are two config options, "excludeRecentSearch" and "excludeOlder" that should be able to suit your needs. They're not in the template (was an oversight) but they exist.~ Edit: that was false.

They're also available as command line options, you can see them with `cross-seed search --help`

**Reply by ajhorvath20** [reply]
Hooray you are the best. 

I will set this like that in my docker-compose `command: search excludeRecentSearch 7200`

**Reply by mmgoodnow** [reply]
Here is the reference for them. CLI options have a different format, they always start with dashes 

https://github.com/mmgoodnow/cross-seed#usage

**Reply by ajhorvath20** [reply]
Thank you so much. Yes that make sense and again thanks
